tests/common/widgets/infinite-hits/options.ts

```diff
@@ -108,88 +106,86 @@ export function createOptionsTests(
       );
     });
 
-    skippableDescribe('banner option', skippedTests, () => {
-      test('renders default banner element with banner widget renderingContent', async () => {
-        const searchClient = createMockedSearchClient({
-          renderingContent: {
-            // @TODO: remove once algoliasearch js client has been updated
-            // @ts-expect-error
-            widgets: {
-              banners: [
-                {
-                  image: {
-                    urls: [{ url: 'https://via.placeholder.com/550x250' }],
-                  },
-                  link: {
-                    url: 'https://www.algolia.com',
-                  },
+    test('renders default banner element with banner widget renderingContent', async () => {
```

Comment:
The snapshot output for the Vue version of this test is adding an extra line between the banner and hit list, but I cannot figure out why. I must be missing something...

https://app.circleci.com/pipelines/github/algolia/instantsearch/11178/workflows/90ab6bc7-58a9-4185-94ab-18d1594139c4/jobs/54699

Reply:
Could this be an undesirable side-effect of prettier applying rules when saving, which might break the generated snapshot? If so, you can either update the snapshot without saving the file again from your editor, or save without formatting being applied.

Reply:
I will double-check, but I don't believe so. The test passes for JS and React, but fails only for Vue.

Reply:
Ah, this then means that the Vue serializer used for Jest outputs the DOM differently then the other ones. To "fix" this, we smooth out differences like this in the normalize function that should be at the top of the file, by usually adding a regular expression that targets the relevant extra whitespace and removes it.

Reply:
Yep, @aymeric-giraudet had just sent me a snippet to try. Worked locally, just pushed another commit.